src/middleware/render-voyager-middleware-page.ts

```diff
@@ -0,0 +1,66 @@
+export interface VoyagerMiddlewareOptions {
+    endpointUrl: string;
+    version: string;
+}
+
+export default function renderVoyagerMiddlewarePage(data: VoyagerMiddlewareOptions) {
+  return `
```

Comment:
@steplov Great 👍 Looking forward to your commit.

> Hope this was the last change.

I reviewed code one more time and don't see any other issues. So after you commit fixes I will merge PR.